common/src/main/java/whocraft/tardis_refined/common/entity/ControlEntity.java

```diff
@@ -22,25 +27,32 @@
 import net.minecraft.world.level.Level;
 import net.minecraft.world.phys.AABB;
 import net.minecraft.world.phys.Vec3;
+import whocraft.tardis_refined.TardisRefined;
+import whocraft.tardis_refined.client.TardisClientData;
 import whocraft.tardis_refined.common.blockentity.console.GlobalConsoleBlockEntity;
 import whocraft.tardis_refined.common.capability.TardisLevelOperator;
 import whocraft.tardis_refined.common.tardis.control.ControlSpecification;
 import whocraft.tardis_refined.common.tardis.control.ship.MonitorControl;
 import whocraft.tardis_refined.common.util.MiscHelper;
+import whocraft.tardis_refined.common.util.Platform;
 import whocraft.tardis_refined.registry.EntityRegistry;
 
 public class ControlEntity extends PathfinderMob {
 
     private ControlSpecification controlSpecification;
     private BlockPos consoleBlockPos;
 
+    private boolean isPointOfInterest = false;
```

Comment:
What is the purpose of this? Looks like it's not being used anywhere.

Reply:
Removed from early experiments.